Makefile.core.mk

```diff
@@ -348,7 +348,8 @@ BINARIES:=./istioctl/cmd/istioctl \
   ./pkg/test/echo/cmd/server \
   ./mixer/test/policybackend \
   ./tools/hyperistio \
-  ./tools/istio-iptables
+  ./tools/istio-iptables \
+  ./tools/istio-clean-iptables
```

Comment:
Where is this used? I don't see it added to any docker files?

Reply:
Would it be simpler to just call `./tools/istio-iptables clean` rather than have a whole separate binary? Likely I am missing some context on the decision to completely split the binary

Reply:
> Where is this used? I don't see it added to any docker files?

It is used in https://github.com/istio/istio/pull/17044/files#diff-1874ac7410fe19e6a97f2410858c36b3R70, added in https://github.com/istio/istio/pull/17044/files#diff-86b940965713741c829f7e4bebf365deR15.

> Would it be simpler to just call ./tools/istio-iptables clean rather than have a whole separate binary

@rlenglet and I think that it is cleaner to move the clean functionality to a separate script/executable.